downloader/src/main/kotlin/Main.kt

```diff
@@ -189,6 +189,7 @@ object Main {
                 }
 
                 log.error { "Could not download '${pkg.id}': ${e.message}" }
+                exitProcess(1)
```

Comment:
Shouldn't we try also for any remaining packages first, and only exit with 1 if there was at least one failure?